.github/workflows/ci.yaml

```
@@ -0,0 +1,17 @@
+name: ci
+
+on: [ push ]
+
+jobs:
+  ci:
+    runs-on: [ ubuntu-latest ]
+    strategy:
+      matrix:
+        target: [ test, lint ]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ~1.17.1
```

If you use "actions/setup-go@v3", you can make this read the Go version from the go.mod file:

```suggestion
    - uses: actions/setup-go@v3
      with:
        go-version-file: go.mod
```